index.html

```diff
@@ -220,6 +220,66 @@ <h3>Multi-Post Stories</h3>
             </div>
           </section>
 
+          <!--Contact-form-->
+          <section id="contact" class="c-form">
+            <p class="con-us">CONTACT US</p>
+            <p class="con-par">
+              If you have an application you are
+              interested in developing, a feature that you
+              need built or a project that needs coding.
+              I'd love to help with it.
+            </p>
+            <form action="https://formspree.io/f/xqknvevr" method="post">
+              <ul class="un-list">
+                <li class="li-item">
+                  <label for="name">
+                    <input type="text" name="name" id="name" class="name-input detail-input" placeholder="Enter your names..."/>
+                  </label>
+                </li>
+                <li class="li-item">
+                  <label for="name">
+                    <input type="email" name="email" id="email" class="email-input detail-input" placeholder="example@gmail.com"/>
+                  </label>
+                </li>
+                <li class="li-item">
+                  <label for="textarea">
+                    <textarea name="text" id="text"  class="text-input detail-input" placeholder="Enter text here..." cols="23" rows="5"></textarea>
+                  </label>
+                </li>
+                <li class="btn-container">
+                  <button type="submit" class="submit-btn">Submit</button>
+                </li>
+              </ul>
```

Comment:
- [ ] Kindly make sure your `Submit` button follows the design like the one from the Figma template to stick to your chosen template. You can check like the: font-family, text color, background color, line-height, padding,...

- [ ]  Also make sure your button follows the interactions such as `hover`, `active` (when it's pressed), `disabled` and enable (default state) are styled as shown in the Figma template. YOu can check all information in the image below from your Figma template.
![btns](https://user-images.githubusercontent.com/83042742/177431814-f1a9867f-fa3e-4eb4-9267-8da7cb66930b.png)